.github/workflows/deploy.yml

```diff
@@ -4,10 +4,13 @@ on:
   # Trigger the workflow every time you push to the `main` branch
   # Using a different branch name? Replace `main` with your branch’s name
   push:
-    branches: [ web ]
```

Comment:
Do we need web branch anymore if we are doing this?

If there is inconsistency between main and stable release , which branch should we use?